packages/browserify/examples/01-simple-js/build.js

```diff
@@ -4,7 +4,11 @@ const browserify = require('browserify')
 // configure LavaMoat
 const lavamoatOpts = {
   writeAutoPolicy: false,
-  scuttleGlobalThis: true,
+  scuttleGlobalThis: {
+    enabled: true,
+    exceptions: ['prompt'],
+    scuttler: 'SCUTTLER',
+  },
```

Comment:
a second afterthought, i know scuttling was introduced before my time ([nov22](https://github.com/LavaMoat/LavaMoat/commit/e90070f2aa8265e5b7e0e90a8d3f3d54f7b0d714#diff-ef5b28753cd110237bb71c073c19be3c94803ad7f42501cd4ae063a96dc0454d))
but since scuttling's an optional and experimental ft
might our simple-js example make more sense stable with scuttling off?
then we could have e.g. `02-simple-js-scuttling` for it on
unless here to promote the bleeding edge :feelsgood:

Reply:
3cec4e1ed